client/my-sites/stats/post-trends/year-navigation.jsx

```diff
@@ -0,0 +1,43 @@
+import { Icon, arrowLeft, arrowRight } from '@wordpress/icons';
```

Comment:
Let's avoid committing code that's not rendered anywhere.

Reply:
@wilmersondasilva, we should remove this file from this PR before moving forward.